contents/handbook/people/hiring-process/engineering-hiring.md

```diff
@@ -9,46 +9,51 @@ hideAnchor: true
 
 Beyond the specific skills listed in the job description, we generally look for: 
 
 *   Experience with relevant technologies (Python or similar, React or similar, something to do with big data is a bonus)
-    *   We don't care how many years of professional experience you have, but depending on our current team structure we may be looking for more or less experienced people for a role - if that's the case, we will be explicit in the job spec  
+    *   We don't care how many years of professional experience you have, but depending on our current team structure we may be looking for more or less experienced people for a role - if that's the case, we will be explicit in the job spec.
 *   Has built something from scratch, ideally with minimal outside help
-    *   You may have been the founder of a startup, or built an impressive side project. You may have also worked on a project at work where you were the only developer. 
+    *   You may have been the founder of a startup, or built an impressive side project. You may have also worked on a project at work where you were the only developer.
 *   Communication skills
     *   More so than other companies, all of our communication is written and public for the world to see. Good written communication is key.
 *   User-centric
-    *   Our engineering team work very closely with our users - they do customer support, demos, and help with implementation. All potential engineers need to be excited by the prospect of getting to work directly with users. 
+    *   Our engineering team work very closely with our users - they do customer support, demos, and help with implementation. All potential engineers need to be excited by the prospect of getting to work directly with users.
 
 ### Engineering hiring process 
 
 #### Culture interview 
 
 This is the usual first round interview with the People & Ops team. 
 
-#### Technical Interview
+#### Technical interview
 
-The [technical interview](/handbook/people/hiring-process#interview-2) round is split into two parts:
+The [technical interview](/handbook/people/hiring-process#2-technical-interview-with-the-hiring-manager) is an hour-long technical interview with one of our engineers. This might be architecture design or diving more into past technical experiences in more of a workshop style. No whiteboarding or brain teasers. 
 
-- The first part is an hour-long technical interview with one of our engineers. This might be architecture design or diving more into past technical experiences in more of a workshop style. No whiteboarding or brainteasers. 
-- If they pass this stage, we will then schedule a follow up 30-minute interview with [Tim](/tim) about your previous experiences and what motivates you. 
+> The engineer interviewing you may be shadowed by another PostHog team member – a shadow is someone who listens in, but doesn't participate. This is something we do regularly among technical interviewers, as a way of improving the hiring process.
```

Comment:
I would say a shadow can participate as well but of course not as active as the main interviewer, this is a way of practicing it/jumping in if you would like to ask something.